File: Cheat.js
export async function requestQuestion(gameId) {
    const apiUrl = `https://corsproxy.io/https://play.kahoot.it/rest/kahoots/${gameId}`;
    const response = await fetch(apiUrl);
    const data = await response.json();
    return data.questions;
}

export function parser(input) {
    const finalDictionary = {};
    input.forEach(question => {
        const questionText = question.question;
        if (questionText) {
            const choices = question.choices || [];
            const correctAnswers = choices
                .filter(option => option.correct === true)
                .map(option => option.answer);
            finalDictionary[questionText] = correctAnswers;
        }
    });
    return finalDictionary;
}

File: Cheat.py
import requests as req
# b84f489e-c4a1-4eed-b185-173caada4fd0

def RequestQuestion():
    GameId = input("Indique o Game-ID")
    r  = req.get(f"https://play.kahoot.it/rest/kahoots/{GameId}") 
    return r.json().get("questions")

def Parser(input):
    FinalDictionary = dict()
    for QuestionNumber in input:
        Questions = QuestionNumber.get("question")
        if Questions is not None:
            Choices = QuestionNumber.get("choices") 
            CorrectAnswers = [
                Option.get("answer") 
                for Option in Choices 
                if Option.get("correct") is True
            ]
            FinalDictionary[Questions] = CorrectAnswers
    return FinalDictionary

print(Parser(RequestQuestion()))